### DISCUSSION TITLE
Add some sort of warning/block to buying items from the rewards catalogue if the account/character in question already has the reward.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5833] [Suggestions]
Currently there is no way to tell from the rewards catalogue if an account (in the case of BoA items/tmog displays) or a character (in the case of BoP items/character services) already has a reward. Because of this, it is exceedingly easy for a player to buy something they already have mistakenly or for a player to purchase a gift reward to another player that already has it. Obviously a solution could just be communication between the players in question or double-checking before purchasing, but the possibility shouldn't even exist, and it would undermine a surprise factor if that's something a player wants to do.

**Comment by Nyeriah** [comment]
We use a third party store plugin to handle the reward store so there's not a lot of leeway to implement new things (we don't control every part of the process) but this is something we should look into.
